Bottom: 9cc86a727a65a813a60b1ad7398c02b479bc28c5
Top:    c26b0b4a7cde86b030fc8186b830143b00f1f559
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-30 11:20:39 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 9785bbe..c19b660 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2474,7 +2474,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
